pkg/watch/watch.go

```diff
@@ -328,7 +328,7 @@ func WatchAndPush(client *occlient.Client, out io.Writer, parameters WatchParame
 
 			deletedPaths = removeDuplicates(deletedPaths)
```

Comment:
shouldn't this be removed as we are doing this already at line 331

Reply:
At line 331, we are removing duplicates between changedFiles and deletedPaths so that if a file is created and deleted or vice versa and watch picks ups both the changes, we just display it once to the user. Here we are removing it from just the deletedPaths.

Reply:
got it